pkg/arm/v9/data/node-startup.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash -ex
 
+# drop all traffic to the kubelet read-only port which doesn't originate from localhost
+# TODO: delete these iptables rules after the azure-monitor team fixes their agent to access the authenticated port
+iptables --insert INPUT --protocol tcp --source localhost --dport 10255 --jump ACCEPT
+iptables --insert INPUT --protocol tcp --dport 10255 --jump DROP
```

Comment:
The order of those two commands should be reversed since it's inserting at the top of the INPUT chain (DROP will end up before ACCEPT).
But this will work because by default IPv6 will be preferred and localhost also points to [::1].

Reply:
@Makdaam Good catch! I actually initially had `--append` in place of `--insert` and switched those sometime before opening this PR to stay consistent with how the rules were specified elsewhere in the codebase. I'll switch these back to `--append`. Thanks!

Reply:
@charlesakalugwu is this the right way around now? happy to lgtm..